content/yaml-quick-start/building-a-cordova-app.md

```diff
@@ -30,7 +30,7 @@ To build, code sign, and publish Cordova Android and iOS apps:
 
 * All Android apps need to be signed before release. See the [Android code signing docs](../code-signing/android-code-signing/) or the sample workflow below for more details.
 * All iOS apps need to be signed before release. See the [iOS code signing docs](../code-signing/ios-code-signing/) or the sample workflow below for more details.
-* All generated artifacts can be published to external services, such as email, Slack, and Google Play. The list of available integrations and script examples can be found in [publishing and deployment](../publishing-yaml/distribution/).
+* All generated artifacts can be published to external services, such as email, Slack, and Google Play. The list of available integrations and script examples can be found under the Publishing section.
```

Comment:
Since we can't link to the category, we could link to the first page in the category, which is the App Store Connect one.